scripts/protocgen.sh

```diff
@@ -23,4 +23,7 @@ done
 # move proto files to the right places
 cp -r github.com/evmos/ethermint/* ./
 rm -rf github.com
-
+mv x/evm/types/params_v0.pb.go x/evm/migrations/v0/types/params_v0.pb.go
```

Comment:
is it possible to do this with the go_package directive in proto file?

Reply:
an [alternative](https://github.com/crypto-org-chain/ethermint/compare/7569bd41f4c9fa3c308d62ae4b7db4c4f5118f62..b728d419b28e97065b6fece6dafe980209e4c17d) way is also wired, any better way to decouple this?